operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -325,7 +332,7 @@ def validateAdminServer(self):
       addError("The admin server '" + adminServerName + "' does not exist.")
       return
     cluster = adminServer.getCluster()
```

Comment:
Do we need a 'WA(' method for 'getCluster()' too?

Reply:
I wondered about that...  That method is all over and we've not seen this use case fail on getCluster().

Reply:
ah, i was trying to work out who "wrapper" became "WA"

Reply:
what about something like getClusterOrNone
or a more generic getOrElse( thingYouWant, None )

Reply:
not a biggie

Reply:
Maybe ignore this comment.  It'd be painful to replace every mbean get in this script with a 'WA(' variant, so maybe we can assume the 'WA' issue only applies to certain mbeans (log, ssl, and dynamic)?   

Any thoughts Johnny? @jshum2479

Reply:
I like the "OrNone" option